app/login/page.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable @next/next/no-img-element */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
build out just the UI, forget about moving the functionality from the base page to the login. it will be done by the inhouse page.